tony-core/src/main/java/com/linkedin/tony/util/Utils.java

```diff
@@ -431,6 +431,17 @@ public static void addResources(String[] resources, Map<String, LocalResource> r
   public static void addResource(String path, Map<String, LocalResource> resourcesMap, FileSystem fs) {
     try {
       if (path != null) {
+        // Check the format of the path, if the path is of path::archive, we set resource type as ARCHIVE
+        String[] pathType = path.split("::");
```

Comment:
This is a preference thing, but how about using `#archive` instead of `::archive`? MR uses `#alias` to support automatically renaming resource files, so I thought using `#archive` seemed more in line with the existing `#` delimiter pattern.

Reply:
makes sense